source/routes/delete/month.js

```diff
@@ -0,0 +1,37 @@
+const PouchDB = require('pouchdb');
+PouchDB.plugin(require('pouchdb-find'));
+const db = new PouchDB('db');
+
+/* *
+Delete month page json object and all bullet documents in that month page.
+*/
+module.exports = {
+  '/delete/month': {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-sp21-group13/cse110-sp21-group13/pullRequest?prid=7423478)
```suggestion
    "/delete/month": {
```